spec/features/tool_drafts/preview/valid/related_urls_preview_spec.rb

```diff
@@ -48,7 +48,7 @@
                 within '.card-body' do
                   expect(page).to have_css('p', text: 'Test related url')
                   expect(page).to have_link(nil, href: 'nasa.gov')
-                  # expect(page).to have_css('li', text: 'GET RELATED VISUALIZATION') # Not rendered... see comments in preview class
+                  expect(page).to have_css('li.arrow-tag-group-item', text: 'GET RELATED VISUALIZATION') #type; uncommented by CT
```

Comment:
I thought that you were going to remove the comment?

Reply:
Fixed.